app/translations/en.json

```
@@ -40,25 +40,30 @@
             "stepFinish": "Finish"
         },
         "initialization": {
-            "pluginsLoadingError": "Plugins information loading error.",
-            "secretsLoadingError": "Secrets information loading error."
+            "pluginsMetadataLoadingError": "Plugins information loading error.",
+            "secretsLoadingError": "Secrets information loading error.",
+            "blueprintsMetadataLoadingError": "Blueprints information loading error"
         },
         "validation": {
             "someTechnologyRequiredError": "At least one technology needs to be selected.",
             "allSecretsRequiredError": "All secret values need to be specified."
         },
         "summary": {
             "taskListTitle": "Task List",
-            "pluginInstalMessageSuffix": "plugin will be installed.",
-            "pluginInstalledMessageSuffix": "plugin is already installed.",
+            "pluginExistsMessageSuffix": "plugin is already installed.",
+            "pluginInstallMessageSuffix": "plugin will be installed.",
             "pluginNotFoundMessageSuffix": "plugin is not found in catalog and manager.",
             "secretCreateMessageSuffix": "secret will be created.",
-            "secretUpdateMessageSuffix": "secret will be updated."
+            "secretUpdateMessageSuffix": "secret will be updated.",
+            "blueprintExistsMessageSuffix": "blueprint is already uploaded.",
+            "blueprintUploadMessageSuffix": "blueprint will be uploaded."
         },
         "installation": {
             "pluginInstallError": "{{pluginName}} plugin installation error.",
             "secretUpdateError": "{{secretName}} secret update operation error.",
             "secretCreateError": "{{secretName}} secret create operation error.",
+            "blueprintUploadError": "{{blueprintName}} blueprint upload error:\n{{uploadError}}",
+            "timeoutExceedError": "Timeout exceed.",
```

Probably more accurate would be:
```suggestion
            "timeoutExceededError": "Timeout exceeded.",
```

---

fixed